core/src/test/java/smile/data/NHANES.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * ***************************************************************************** Copyright (c)
+ * 2010-2019 Haifeng Li
+ *
+ * <p>Smile is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version.
+ *
+ * <p>Smile is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * <p>You should have received a copy of the GNU Lesser General Public License along with Smile. If
+ * not, see <https://www.gnu.org/licenses/>.
+ * *****************************************************************************
+ */
+package smile.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.csv.CSVFormat;
+
+import smile.data.formula.Formula;
+import smile.data.type.DataTypes;
+import smile.data.type.StructField;
+import smile.data.type.StructType;
+import smile.io.CSV;
+import smile.util.Paths;
+
+/**
+ * This data is from NHANES I with followup mortality data of the NHANES I Epidemiologic Followup
+ * Study.
+ *
+ * <pre>https://wwwn.cdc.gov/nchs/nhanes/nhanes1</pre>
+ *
+ * <pre>https://wwwn.cdc.gov/nchs/nhanes/nhefs</pre>
+ *
+ * @author ray
+ */
+public class NHANES {
+
+  public static DataFrame data;
+  public static Formula formula = Formula.lhs("y");
+
+  public static double[][] x;
+  public static double[] y;
+
+  static {
+    List<StructField> fields = new ArrayList<StructField>();
+    fields.add(new StructField("y", DataTypes.DoubleType));
```

Comment:
not necessary. Read.csv will infer schema.

Reply:
> not necessary. Read.csv will infer schema.

inferred schema will have generated field name like V1/V2/...? thought it would be more straight-forward to add schema explictly in this dataset specific class.

Reply:
The header line will be used for variable names.